resources/etc/supervisord.conf

```diff
@@ -1,4 +1,5 @@
 [supervisord]
+user = root
```

Comment:
Is this necessary? This could prevent us when we apply the DoD changes, including performing everything as a non-root user